cypress/e2e/playground.cy.ts

```diff
@@ -0,0 +1,87 @@
+const getIframeBody = () => {
+    return cy
+      .get('iframe[id="FCL_IFRAME"]')
+      .its('0.contentDocument.body').should('not.be.empty')
+      .then(cy.wrap)
+  }
+  
+describe('Flow-Playground frontend tests', () => {
+
+    beforeEach(() => {
+        cy.viewport('macbook-16')
+        cy.visit('http://localhost:3000/')
```

Comment:
Can we use a [baseUrl](https://docs.cypress.io/guides/references/best-practices#Setting-a-global-baseUrl) option?